internal/pkg/handler/updated-handler.go

```diff
@@ -236,60 +234,8 @@ func updateContainers(containers []v1.Container, annotationValue string, sshData
 			}
 			containers[i].Env = append(containers[i].Env, e)
 			updated = true
-			logrus.Infof("%s environment variable does not found so creating a new one")
+			logrus.Infof("%s environment variable does not found, creating a new env with value %s", envar, sshData)
```

Comment:
Change the text to "environment variable not found" or "environment variable does not exist". 

Plus we should log the name of the object i.e. deployment daemonset, stateful set etc instead of the SHA value